package.json

```diff
@@ -38,6 +38,7 @@
     "debug": "2.3.2",
     "double-ended-queue": "2.1.0-0",
     "fruitdown": "1.0.2",
+    "immediate": "3.0.6",
```

Comment:
We use an older version of `immediate` because it's smaller and it's what `lie` uses. This is explained in a comment in `nextTick-browser.js`.